src/openpersonen/contrib/stufbg/converters/ingeschreven_persoon.py

```diff
@@ -90,8 +93,8 @@ def get_persoon_instance_dict(response, instance_xml_dict, prefix):
             },
             "land": {
                 "code": instance_xml_dict.get(f"{prefix}:inp.geboorteLand", "string"),
-                "omschrijving": instance_xml_dict.get(
-                    f"{prefix}:inp.geboorteLand", "string"
+                "omschrijving": CountryCodeAndOmschrijving.get_omschrijving_from_code(
+                    instance_xml_dict.get(f"{prefix}:inp.geboorteLand", "string")
```

Comment:
Not very important, but since `0` is used as the default everywhere else, I'd expect that here too
```suggestion
                    instance_xml_dict.get(f"{prefix}:inp.geboorteLand", "0")
```